python/ray/rllib/models/catalog.py

```diff
@@ -75,7 +76,11 @@ def ConvolutionalNetwork(inputs, num_outputs, options=None):
         return ConvolutionalNetwork(inputs, num_outputs, options)
 
     @staticmethod
-    def get_preprocessor(env_name, obs_shape):
+    def LSTM(inputs, num_outputs, options=None):
```

Comment:
Sounds good. It would also be nice to figure out if/how to get LSTM support for more algorithms, if it provides a substantial performance increase.